test/testsuites/examples/create-eks.yaml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-eks-task
```

Comment:
Is it possible to use generate name for uniqueness?

Reply:
Tasks are only run with PipelineRuns or TaskRuns. When the tasks are created as pods with PipelineRuns, they're pre-pended with the name of the pipeline like `{pipeline_run_name}-{name_gen_characters}-{task-name}`